charts/nginx-gateway-fabric/values.yaml

```diff
@@ -592,6 +601,16 @@ nginx:
     # - port: 30025
     #   listenerPort: 80
 
+    # -- Custom patches to apply to the NGINX Service.
+    patches: []
+    # -- Example:
+    # patches:
+    #   - type: JSONPatch
+    #     value:
+    #       - op: replace
+    #         path: /spec/externalTrafficPolicy
+    #         value: "Cluster"
```

Comment:
nit: I'd use a different example here since this field is exposed natively in our API